Library/OcDebugLogLib/OcLog.c

```diff
@@ -522,6 +687,18 @@ OcConfigureLogProtocol (
       Private->OcLog.FileSystem   = LogRoot;
       Private->OcLog.FilePath     = LogPath;
 
+      //
+      // Write filters into Private.
+      //
+      FlexFilters = NULL;
+      if (AsciiStrCmp (LogModule, "*") != 0 && AsciiStrCmp (LogModule, "") != 0) {
+        LogModuleCopy = AllocateCopyPool (AsciiStrSize (LogModule), LogModule);
+        ASSERT (LogModuleCopy != NULL);
+        FlexFilters   = OcStringSplit (LogModuleCopy, L',', FALSE);
+        ASSERT (FlexFilters != NULL);
```

Comment:
Probably also needs to be a runtime check, or what is the reason this cannot fail?

Reply:
https://github.com/acidanthera/OpenCorePkg/commit/486dcc95f0f3f1202951d7a8aa7a32a99bffcf22